utils/staking-miner/src/token.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use ss58_registry::{Token as WrappedToken, TokenAmount};
+
+static mut TOKEN: WrappedToken = TokenRegistry::Dot.into();
+
+pub struct Token;
+
+impl Token {
+	pub fn init(token: WrappedToken) {
```

Comment:
I would rather mark these functions as unsafe because it really depends how it's used thus the type itself can't ensure that's is single threaded or not.